### ISSUE TITLE
Building add-on causes ENOENT, no such file or directory

### ISSUE THREAD

**Issue by bradurani** [ember-cli/ember-cli#3059]
When running ember build for my add-on I'm getting:

`Build failed.
ENOENT, no such file or directory '/path/to/my/addon/tmp/funnel-dest_J6Xen7.tmp/index.html'`

My `package.json` looks like this:

{
  "name": "my-add-on",
  "version": "0.0.0",
  "description": "The default blueprint for ember-cli addons.",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "scripts": {
    "start": "ember server",
    "build": "ember build",
    "test": "ember test"
  },
  "repository": "",
  "engines": {
    "node": ">= 0.10.0"
  },
  "author": "",
  "license": "MIT",
  "devDependencies": {
    "broccoli-asset-rev": "^2.0.0",
    "broccoli-ember-hbs-template-compiler": "^1.6.1",
    "ember-cli": "0.1.7",
    "ember-cli-6to5": "0.2.1",
    "ember-cli-content-security-policy": "0.3.0",
    "ember-cli-dependency-checker": "0.0.7",
    "ember-cli-ic-ajax": "0.1.1",
    "ember-cli-inject-live-reload": "^1.3.0",
    "ember-cli-qunit": "0.1.2",
    "ember-data": "^1.0.0-beta.12",
    "ember-export-application-global": "^1.0.0",
    "express": "^4.8.5",
    "glob": "^4.0.5"
  },
  "keywords": [
    "ember-addon"
  ],
  "ember-addon": {
    "configPath": "tests/dummy/config"
  }
}

When I run `npm update --save-dev`
I get:

`npm ERR! cb() never called!`

node version: `v0.10.33`
npm version: `2.2.0`
Not sure what to do. Any help appreciated

**Comment by bradurani** [comment]
Update: I was able to fix the `npm ERR! cb() never called!` by upgrading node and nom, but still experiencing the primary problem

**Comment by stefanpenner** [comment]
i believe this is related to the fix @jgwhite got into master. We should likely release again in the next day

@jgwhite can you confirm?

**Comment by jgwhite** [comment]
Confirm. The npmignore fix should address this.

**Comment by stefanpenner** [comment]
Fixed on master, we will get a release out by friday

**Comment by bradurani** [comment]
Thanks!

**Comment by johnotander** [comment]
@bradurani I just ran into this issue as well. I'm not sure if you were having issues with an ember addon you've already generated, or if you were just reporting the fact that the addon generator is a bit broken on `0.1.7`.

If it's the former, I was able to get my already generated addon build working by changing

```js
var EmberApp = require('ember-cli/lib/broccoli/ember-app');
```

to

```js
var EmberApp = require('ember-cli/lib/broccoli/ember-addon');
```

in my Brocfile.js. Just in case anyone stumbles across this issue before the next version is cut.
